requirements.txt

```diff
@@ -2,3 +2,5 @@ pytest<8.0
 filecheck<0.0.23
 lit<16.0.0
 frozenlist<1.4.*
+codecov
```

Comment:
Should this be in `requirements-optional.txt instead?

Reply:
Yes, moving.